src/process.c

```diff
@@ -109,10 +109,12 @@ static pid_t ebpf_update_tables(int global, int apps)
                                         .create_process = 1, .create_thread = 1, .task_err = 1, 
                                         .removeme = 0 };
 
-    uint32_t idx = (uint32_t)pid;
-    int ret = bpf_map_update_elem(apps, &idx, &stats, 0);
-    if (ret)
-        fprintf(stderr, "Cannot insert value to global table.");
+    uint32_t idx;
+    for (idx = 0 ; idx < 10; idx++) {
```

Comment:
Magic number? What it is and why do you use it here?

Reply:
Yes, it is almost this. I need to have something inside the table to read it with other functions, so I am only storing data, to be sure that it will be there to be read.
Another possible approach would be to start different software to store data, but depending of the selected PID collection, this could create only one entry inside the tables.

Reply:
Please add a macro with a relevant name for the number.

Reply:
Last commit added the macro.